colabxterm/notebook.py

```diff
@@ -76,6 +76,26 @@ def is_port_in_use(port: int) -> bool:
     parsed_args = shlex.split(args_string, comments=True, posix=True)
     height = 800
     port = 10000
+
+    # setup parameter from parsed_args to support the following format
+    # %xterm height=300 port=400
+    for parameter in parsed_args:
+      kv_pair:list[str] = str(parameter).split('=')
+      if(len(kv_pair)==2):
```

Comment:
if len(kv_pair)==2:

Reply:
Thanks.
Solved in the latest commit.   https://github.com/InfuseAI/colab-xterm/pull/11/commits/ede68bd19762375b46d46587f7c5bad6c7c13a46